backend/tests/test_auth.py

```
@@ -0,0 +1,108 @@
+        yield db
+    finally:
+        db.close()
+
+# Override DB session dependency globally
+app.dependency_overrides[Session] = override_get_db
+
+# ----------------------
+# 🔧 Fixture for test user
+# ----------------------
+@pytest.fixture
+def test_user():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_user`

(ANN201)